dotnetv3/Polly/SynthesizeSpeechExample/SynthesizeSpeech.cs

```diff
@@ -64,17 +62,24 @@ private static async Task<SynthesizeSpeechResponse> PollySynthesizeSpeech (IAmaz
         /// save the audio stream.</param>
         private static void WriteSpeechToStream(Stream audioStream, string outputFileName)
         {
-            var outputStream = new FileStream(outputFileName, FileMode.Create,
+            var outputStream = new FileStream(
+                outputFileName,
+                FileMode.Create,
                 FileAccess.Write);
             byte[] buffer = new byte[2 * 1024];
             int readBytes;
 
             while ((readBytes = audioStream.Read(buffer, 0, 2 * 1024)) > 0)
+            {
                 outputStream.Write(buffer, 0, readBytes);
+            }
 
             // If we don't flush the buffer, we lose the last second or so of
             // the synthesized text.
```

Comment:
Flushes the buffer to avoid losing the last second or so of the synthesized text.